src/order/complete_lattice.lean

```diff
@@ -1072,6 +1072,72 @@ instance pi.complete_lattice {α : Type*} {β : α → Type*} [∀ i, complete_l
   le_Inf := λ s f hf i, le_infi $ λ g, hf g g.2 i,
   .. pi.bounded_lattice }
 
+/- The Knaster-Tarski theorem -/
+def complete_lattice_fixed_points {α : Type*} [complete_lattice α] {f : α → α} (hf : monotone f) :
+complete_lattice {x // x = f x} :=
```

Comment:
I think this might make sense to state as
```suggestion
complete_lattice (function.fixed_points f) :=
```
which would suggest this belongs in a new `dynamics/fixed_points/lattice.lean` or similar

Reply:
Oh, oops, look at this: [https://leanprover-community.github.io/mathlib_docs/order/fixed_points.html#fixed_points.complete_lattice](url)

Reply:
Well you could at least add `Knaster-Tarski` to the docstring of that one, and quite possibly golf some proofs